src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -187,11 +188,7 @@ public RWRoute(Design design, RWRouteConfig config){
 		
 		RoutableNode.setMaskNodesCrossRCLK(this.config.isMaskNodesCrossRCLK());
 
-		if(this.config.isTimingDriven()) {
-			DSPTimingData.setDSPTimingFolder(config.getDspTimingDataFolder());
-			ClkRouteTiming clkTiming = createClkTimingData(config);
-			this.routesToSinkINTTiles = clkTiming == null? null : clkTiming.getRoutesToSinkINTTiles();
-			this.timingManager = new TimingManager(this.design, true, this.routerTimer, this.config, clkTiming);		
+		if(this.config.isTimingDriven()) {		
```

Comment:
I think this issue got overlooked.  Please take a look at the link, I see lots of uses of `this` that are unnecessary.

Reply:
Thanks for the reminder, Chris. It is because using this allows intellisense to pop up in Eclipse that I used this a lot. I was unware that they are supposed to be heavy. 
Those uses of this have been removed in the updated files.

Reply:
Thanks @YunxZhou for cleaning up the `this` references.  In Eclipse, you can always bring up the Intellisense menu by using the keyboard shortcut `<Ctrl>+<Space>`.  I use it all the time and avoids having to type `this` all the time.